a_tour_of_textile.yml

```diff
@@ -0,0 +1,472 @@
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+    ls_invites:
+      cmd:
+        code: 'textile invites ls --api="http://127.0.0.1:41600"'
+      js_http_client:
+        code: 'Textile.invites.list()'
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+    accept:
+      cmd:
+        code: 'textile invites accept "QmXcJmyX2vbeJTcZSkoZCHc74yycjJcXbxCHkLknJhyPaL" --api="http://127.0.0.1:41600"'
+      js_http_client:
+        code: 'Textile.invites.accept("QmXcJmyX2vbeJTcZSkoZCHc74yycjJcXbxCHkLknJhyPaL")'
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+    blocks_ls:
+      cmd:
+        code: 'textile blocks ls --thread="12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw"'
+      js_http_client:
+        code: 'Textile.blocks.list("12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw")'
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+    comment_add:
+      cmd:
+        code: 'textile comments add "Is this an outlier?" --block="QmUvWjstQzR6y7UctRJgVjcKsKzutZoiBsQw6WBXMnmg84" --api="http://127.0.0.1:41600"'
+      js_http_client:
+        code: 'Textile.comments.add("QmUvWjstQzR6y7UctRJgVjcKsKzutZoiBsQw6WBXMnmg84", "Is this an outlier?")'
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+    like_add:
+      cmd:
+        code: 'textile likes add --block="QmVoKpKsg5MkW11bK3LVmd3xMMaxTutVime32sV6EZWeLk"'
+      js_http_client:
+        code: 'Textile.likes.add("QmUvWjstQzR6y7UctRJgVjcKsKzutZoiBsQw6WBXMnmg84")'
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+    files_ls:
+      cmd:
+        code: 'textile files ls --thread="12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw"'
+      js_http_client:
+        code: 'Textile.files.list("12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw")'
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+    feed:
+      cmd:
+        code: 'textile feed --thread="12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw"'
+      js_http_client:
+        code: 'Textile.feed.get("12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw")'
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+    add_location_data:
+      cmd:
+        code: "echo '{ \"latitude\": 48.858093, \"longitude\": 2.294694 }' | textile files add --thread=\"12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw\"'"
+      js_http_client:
+        code: 'Textile.files.add(JSON.stringify({ \"latitude\": 48.858093, \"longitude\": 2.294694 }), "", "12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw")'
```

Comment:
We should just support adding JSON/JS objects directly here. No need to stringify on the part of the user. Will create a ticket for this as well.